### PR TITLE
AutoYaST SLE 15 documentation updates

### PR DESCRIPTION
**Pull request by imobachgs**
Small updates to AutoYaST documentation for SLE/openSUSE 15.

### CODE REVIEW

File: xml/ay_bigfile.xml [hunk 1]
```diff
@@ -2925,14 +2929,9 @@ openssl x509 -noout -fingerprint -sha256
...
-          partition should be resized. In this case, you want to set
-          <literal>create</literal> to <literal>false</literal> and usually
-          you do not want to <literal>format</literal> the partition.  You need
-          to tell &ay; the <literal>partition_nr</literal> and the
-          <literal>size</literal>. The size can be in percentage of the
-          original size or a number, like <literal>800M</literal>.
-          <literal>max</literal> and <literal>auto</literal> do not work as
-          size here.
+          partition should be resized. In this case, you need to tell &ay;
+          the reuse the device (see the <literal>create</literal>) and specify
```
Comment (svenseeberg) [inline xml/ay_bigfile.xml]: "the reuse" -> "to reuse"?

File: xml/ay_bigfile.xml [hunk 2]
```diff
@@ -2946,6 +2945,17 @@ openssl x509 -noout -fingerprint -sha256
...
      </tgroup>
     </informaltable>
+
+    <important>
+     <title>Boot Partition Might not Be Created</title>
+     <para>
+      If <literal>size</literal> is set to <literal>auto</literal> for the
+      <literal>boot</literal> partition, &ay; will not create it unless is
+      strictly required in order to boot. If you want to force the creation of
+      such a partition, you should specify the size using a number or a
```
Comment (svenseeberg) [inline xml/ay_bigfile.xml]: just a suggestion: "such a partition" -> "boot partition"

**Comment by imobachgs** [comment]
@svenseeberg Thanks for the review. I've updated the PR.

**Comment by svenseeberg** [comment]
Thanks for the PR :-)

**Comment by imobachgs** [comment]
Thanks for the review :) More to come in a new future... :)

**Pull request merged**